src/ui/interface.js

```diff
@@ -853,7 +853,33 @@ var UI = Class.create({
 		} else {
 			this.closeDash();
 		}
+	},
 
+	godletToggle: function() {
```

Comment:
`toggleGodlet` would be a better function name.

I haven't tested the patch yet, but as far as I can tell, this might have been approached better as simply having a random bolean parameter for `toggleDash` function, because there are quite a few other UI fixes/patches that rely on toggleDash function itself. Gonna comment on the PR after testing anyway.